ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-stable-prod.yaml

```diff
@@ -0,0 +1,39 @@
+build_root:
+  image_stream_tag:
+    name: ci-tools-build-root
+    namespace: ci
+    tag: latest
+images:
+- dockerfile_path: prow/Dockerfile
+  to: ocp-qe-perfscale-ci-rosa-4.12-prod.yaml
+promotion:
+  namespace: ci
+  tag: latest
+releases:
+  latest:
+    release:
+      channel: candidate
+      version: "4.12"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: ocp-qe-perfscale-ci-tests-rosa-hypershift-cluster-density
+  cron: 0 12 * * 5
+  steps:
+    cluster_profile: aws-perf-qe
+    env:
+      OCM_LOGIN_ENV: production
+      OPENSHIFT_VERSION: "4.12"
+      REGION: us-east-2
+      ROSA_LOGIN_ENV: production
+    test:
+    - ref: openshift-qe-cluster-density
+    workflow: rosa-aws-sts-hypershift
```

Comment:
Should we put hypershift in the file name?

Reply:
Good point. I will update it.

Reply:
On a second thought, thinking that we may want to pull all prod tests in this file instead of one for each workload type or architecture type.

The only tests I expect to run on prod are reliability tests and cluster-density as sanity test, whether Hypershift or Rosa or non-Rosa. All other tests will be in stage anyway. 

Let me know if that makes sense.